src/lib/DriveProvider.jsx

```diff
@@ -25,14 +25,16 @@ const DriveProvider = ({ client, lang, polyglot, dictRequire, children }) => {
         <VaultProvider cozyClient={client}>
           <VaultUnlockProvider>
             <SharingProvider doctype="io.cozy.files" documentType="Files">
-              <CozyTheme ignoreCozySettings={isPublic} className="u-w-100">
-                <BreakpointsProvider>
-                  <AlertProvider>
-                    <VaultUnlockPlaceholder />
-                    <FabProvider>{children}</FabProvider>
-                  </AlertProvider>
-                </BreakpointsProvider>
-              </CozyTheme>
+              <NativeFileSharingProvider>
```

Comment:
Pas besoin de scope au contexte "privé" (non public/intent) ?

Reply:
@Ldoppea Est-ce que c'est possible d'ouvrir des dossiers publics type partage par lien dans l'AA ?

Reply:
On a rien de spécifiquement prévu pour en tout cas. Lorsqu'on ouvre un lien, s'il est sur un domaine autre que le cozy connecté, alors on ouvre le lien tel quel dans un in-app browser (donc pas d'injection de cookies ni rien). S'il est sur un même domaine (genre si tu ouvre ton propre lien public), probablement que ça va ouvrir la webview cozy-app sur drive de façon classique (donc avec tous les mécanismes de httpservers + cookies & co), mais je n'ai pas en tête le comportement exacte. Le mieux serait d'essayer.

Reply:
J'ai fait quelques tests sans arriver à des cas problématiques. 

Pour résumer si on ouvre un inapp browser (donc les partages par lien et les partages cozy à cozy pas encore acceptés), alors on est pas dans l'AA, alors l'action ne s'affiche pas.

Si on accepte à un dossier partagé qui a été synchronisé et qui s'affiche donc dans "notre" drive, là tout marche normalement.